app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    dict_ = {}
+
+    def inner(*args) -> Callable:
```

Comment:
what if function take key arguments
```suggestion
    def inner(*args, **kwargs) -> Callable:
```